scripts/scripts.js

```diff
@@ -427,6 +427,23 @@ function decorateLinks(main) {
     }
   });
 }
+
+function decorateImages(main) {
+  // Get all img elements within the main container
+  const images = main.querySelectorAll('img');
+
+  // Get the first image and set class to hero-image
+  const firstImage = images[0];
+  if (firstImage) {
+    firstImage.classList.add('hero-image');
```

Comment:
I wonder if it would not be best to add the class on the parent paragraph, rather than the image. WDYT? Not sure, so this is not a recommendation, rather a wondering. 🙂

Reply:
I prefer it on the images since it is more targeted. We can add the class on paragraph and then specify the img on styling but we don't gain anything.